### PR TITLE
add neutral gas data comparison plots for individual galaxies

### PR DESCRIPTION
**Pull request by wmfw23**
following data added to the observational_data repository in [PR:75](https://github.com/SWIFTSIM/velociraptor-comparison-data/pull/75)

### CODE REVIEW

File: colibre/auto_plotter/cold_gas_fractions.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    start:
+      value: 2e7
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Cold gas Fraction
+    section: Cold Gas Data Comparison
+    caption: Galaxy H_$2$ mass (including Helium following the observations) over stellar mass as a function of stellar mass in 0.2 dex bins, measured in 100 kpc apertures
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/cold_gas_fractions.yml:29]: Galaxy H_$2$  -> Galaxy H$_2$?

Also, it looks like the bins here are not 0.2 dex

File: colibre/auto_plotter/cold_gas_fractions.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    start:
+      value: 1e-4
+      units: "1 / Gigayear"
+    end:
+      value: 6e1
+      units: "1 / Gigayear"
+  metadata:
+    title: sSFR-H2 Fraction
+    section: Cold Gas Data Comparison
+    caption: Galaxy H_$2$ mass (including Helium following the observations) over neutral gas mass (HI+H$_2$) as a function of specific star formation rate in 0.2 dex bins, measured in 100 kpc apertures
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/cold_gas_fractions.yml:125]: -- Galaxy H_$2$ -> Galaxy H$_2$?
-- The comment says the bins here are 0.2 dex-wide but I don't think that is the case

File: colibre/auto_plotter/cold_gas_fractions.yml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    start:
+      value: 2e7
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Cold gas Fraction
+    section: Cold Gas Data Comparison
+    caption: Galaxy H_$2$ mass (including Helium following the observations) over stellar mass as a function of stellar mass in 0.2 dex bins, measured in 100 kpc apertures
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/cold_gas_fractions.yml:93]: You are plotting H2-to-neutral H fraction but the caption says H2 over stellar mass

File: colibre/auto_plotter/cold_gas_fractions.yml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    start:
+      value: 1e-4
+      units: "1 / Gigayear"
+    end:
+      value: 6e1
+      units: "1 / Gigayear"
+  metadata:
+    title: sSFR-Cold gas Fraction
+    section: Cold Gas Data Comparison
+    caption: Galaxy H_$2$ mass (including Helium following the observations) over stellar mass as a function of specific star formation rate in 0.2 dex bins, measured in 100 kpc apertures
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/cold_gas_fractions.yml:61]: Same comment about H_$2$ and bin size

**Review by EvgeniiChaikin — CHANGES_REQUESTED** [review]
Hi @wmfw23 I guess this PR goes together with your most recent PR to the comparison-data repository? 

If so, I suggest

1.  we first update the observational data repo to include your new files;
2.  and then proceed with this PR.

After step one is done, It would also be great if you could attach the observational-data submodule to the most recent commit so that it includes your new data.

**Comment by wmfw23** [comment]
Thanks Evgenii, yes once [This PR](https://github.com/SWIFTSIM/velociraptor-comparison-data/pull/75) is integrated into the main branch, I can update the submodule here to the main branch and push a commit

File: colibre/auto_plotter/cold_gas_fractions.yml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  y:
+    quantity: "derived_quantities.gas_H2_to_neutral_H_fraction_100_kpc"
+    units: "Solar_Mass / Solar_Mass"
+    start: 1e-4
+    end: 1
+  median:
+    plot: true
+    adaptive: true
+    log: true
+    number_of_bins: 30
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/cold_gas_fractions.yml:83]: @wmfw23 Can you change it to 25 and update the submodule, and then we merge it?

**Review by EvgeniiChaikin — APPROVED** [review]

**Comment by JBorrow** [comment]
Please rebase before merging

**Comment by JBorrow** [comment]
Thanks both for your help!

**Pull request merged**